source/_static/scss/includes/_aside.scss

```diff
@@ -1,4 +1,4 @@
-.sidebar {
+div.sidebar,aside.sidebar {
```

Comment:
@ravind It seems like if we remove the element ref from the class names, it would fix the style issues. So we can keep it simple as before `.sidebar` { ... }

Reply:
Dropping `div.sidebar` seems to be fine, but `aside.sidebar` appears necessary.